sharded_queue/api.lua

```diff
@@ -319,24 +292,28 @@ function sharded_queue.statistics(tube_name)
             leader_only = true
         })
 
-    local ok, stats_collection = map('tube_statistic',
-        { tube_name = tube_name }, storages)
+    local stats_collection, err = cartridge_pool.map_call('tube_statistic',
+        {{ tube_name = tube_name }}, {uri_list=storages})
 
-    if not ok or stats_collection == nil then
-        log.info(stats_collection)
-        return
+    if err then
```

Comment:
not stats_collection or err ~= nil

Reply:
`err = err or 'No stats retrieved'` or a better error description

Reply:
Checked https://github.com/tarantool/sharded-queue/pull/19/commits/9c3cccb6371bfa6b743c8c11933af3872762e7da